readme.html

```diff
@@ -53,5 +62,124 @@ <h2>Pagination</h2>
+            </tr>
+        </table>
+    </p>
+    <p>
+        <h2>Permissions</h2>
+        <p>
+            Super admins can assign admin permissions to individuals or groups.
+            <br>
+            <b>Individual</b> assignments are better for a manageable number of admin accounts. When you assign admin permissions to individuals, you do so one at a time, whenever necessary. Remember that every organisation must always have at least one individual assigned super admin.
+            <br>
+            <b>Admin groups</b> work better when you need to onboard a large number of admins quickly. Assign those admins to one group, and then grant admin permissions to that group. Okta groups, AD groups, and LDAP groups are all eligible.
+            <br><br>
+            To assign the roles to any user, it is required to run the <b>"assign role"</b> action using the asset configured with the API token of a <b>Super admin user(SUPER_ADMIN)</b> in the OKTA server to assign appropriate permissions to that normal user.
+            <br><br>
+            Step to assign a Super admin role to user or group.
+            <br>
+            <ul>
+                <li>1. In the Admin Console, go to <b>Security > Administrators</b>.
+                </li>
+                <li>
+                    2. Click <b>Add Administrator</b>, depending on whether you are assigning privileges to an individual or group.
+                </li>
+                <li>
+                    3. In the <b>Grant administrator role to</b> field, begin typing the name of the user that we want to assign admin privileges to and select the correct user from the search list.
+                </li>
+                <li>
+                    4. Select the <b>Super Administrator</b> roles you want the user or group to have. You can assign multiple admin roles to an individual or group.
+                </li>
+                <li>
+                    5. Some admin roles require additional input. If you selected Application Administrator, Group Administrator, Help Desk Administrator, Group Membership Administrator, you need to indicate whether that role administers all users, groups, apps, just specific ones. (Copy and Paste commands allow you to apply the same assignments to multiple roles).
+                </li>
+                <li>
+                    6. Click Add Administrator to complete the assignment.
+                </li>
+            </ul>
+            <br>
+        </p>
+        <p>
+            The following tables show the Details of the roles and what they can perform.
+            <table>
+              <tr>
+                <td>Role type</td>
+                <td>Label</td>
+                <td>Detail</td>
+              </tr>
+              <tr>
+                <td>SUPER_ADMIN</td>
+                <td>Super Administrator</td>
+                <td>Perform all admin activities for an organisation. Super admins have full management access.</td>
```

Comment:
```suggestion
                <td>Perform all admin activities for an organization. Super admins have full management access.</td>
```

Reply:
Changed word from `organisation` to `organization`.